control/statesp.py

```diff
@@ -679,16 +681,26 @@ def _convertToStateSpace(sys, **kw):
                 ssout[3][:sys.outputs, :states],
                 ssout[4], sys.dt)
         except ImportError:
-            # If slycot is not available, use signal.lti (SISO only)
-            if (sys.inputs != 1 or sys.outputs != 1):
-                raise TypeError("No support for MIMO without slycot")
-
-            # TODO: do we want to squeeze first and check dimenations?
-            # I think this will fail if num and den aren't 1-D after
-            # the squeeze
-            lti_sys = lti(squeeze(sys.num), squeeze(sys.den))
-            return StateSpace(lti_sys.A, lti_sys.B, lti_sys.C, lti_sys.D,
-                              sys.dt)
+            # No Slycot.  Scipy tf->ss can't handle MIMO, but static MIMO is an easy special case we can check for here
+            maxn = max(max(len(n) for n in nrow)
+                       for nrow in sys.num)
+            maxd = max(max(len(d) for d in drow)
+                       for drow in sys.den)
+            if 1==maxn and 1==maxd:
+                D = empty((sys.outputs,sys.inputs),dtype=float)
+                for i,j in itertools.product(range(sys.outputs),range(sys.inputs)):
+                    D[i,j] = sys.num[i][j][0] / sys.den[i][j][0]
+                return StateSpace([], [], [], D, sys.dt)
+            else:
+                if (sys.inputs != 1 or sys.outputs != 1):
+                    raise TypeError("No support for MIMO without slycot")
+
+                # TODO: do we want to squeeze first and check dimenations?
```

Comment:
I can't take the credit for this one, the comment was there already ;).  Not sure it's applicable anymore: the check that sys.inputs and sys.outputs are both 1 implies that squeeze *will* bring num and den to 1-D?